lib/puma/cli.rb

```diff
@@ -157,6 +157,10 @@ def setup_options
             user_config.pidfile arg
           end
 
+          o.on "--plugin PLUGIN", "Loads puma PLUGINs" do |arg|
```

Comment:
```suggestion
          o.on "--plugin PLUGIN", "Load the given PLUGIN. Can be used multiple times to load multiple plugins." do |arg|
```

Reply:
Thank you for the review @dentarg ! Applied the suggested changes.